package.json

```diff
@@ -54,6 +54,7 @@
     "moment": "2.29.4",
     "moment-duration-format": "2.3.2",
     "moment-timezone": "0.5.41",
+    "nasa-grammar": "1.1.1",
```

Comment:
doesn't this introduce a potential risk like left-pad ([example](https://www.theregister.com/2016/03/23/npm_left_pad_chaos/)) of a random minimal 3rd party library . Couldn't the custom dictionary just be added in the codebase [doc](https://cspell.org/docs/dictionaries-custom/)

Reply:
@depperm in this case, the npm package is owned by me and I'm happy to hand it over to you. My idea was that we group all nasa repository dictionaries into one package, where each team would have either a namespaced dictionary like `nasa-openmct` and `nasa-ammos`. For now it's just `nasa` but if you would like to use cspell in other js/ts projects, they're under the same umberella. 

The package itself doesn't have any dependencies, so won't need to worry about left-pad